### ISSUE TITLE
npm install osrm fails on s3 download

### ISSUE THREAD

**Issue by dfellis** [Project-OSRM/osrm-backend#4675]
I just searched the issues for "npm" and didn't find anything relevant. It looks like the osrm tarball has been (accidentally) made private so `npm install osrm` does not work.

```sh
$ npm install --save osrm

> osrm@5.13.0 install /home/damocles/cardash/osrm-test/node_modules/osrm
> node-pre-gyp install --fallback-to-build=false || ./scripts/node_install.sh

node-pre-gyp ERR! install error
node-pre-gyp ERR! stack Error: 403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/osrm/v5.13.0/Release/node-v57-linux-x64.tar.gz
node-pre-gyp ERR! stack     at Request.<anonymous> (/home/damocles/cardash/osrm-test/node_modules/osrm/node_modules/node-pre-gyp/lib/install.js:118:27)
node-pre-gyp ERR! stack     at emitOne (events.js:120:20)
node-pre-gyp ERR! stack     at Request.emit (events.js:210:7)
node-pre-gyp ERR! stack     at Request.onRequestResponse (/home/damocles/cardash/osrm-test/node_modules/osrm/node_modules/request/request.js:1074:10)
node-pre-gyp ERR! stack     at emitOne (events.js:115:13)
node-pre-gyp ERR! stack     at ClientRequest.emit (events.js:210:7)
node-pre-gyp ERR! stack     at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:549:21)
node-pre-gyp ERR! stack     at HTTPParser.parserOnHeadersComplete (_http_common.js:116:23)
node-pre-gyp ERR! stack     at TLSSocket.socketOnData (_http_client.js:438:20)
node-pre-gyp ERR! stack     at emitOne (events.js:115:13)
node-pre-gyp ERR! System Linux 4.13.0-16-generic
node-pre-gyp ERR! command "/home/damocles/.nvm/versions/node/v8.7.0/bin/node" "/home/damocles/cardash/osrm-test/node_modules/osrm/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build=false"
node-pre-gyp ERR! cwd /home/damocles/cardash/osrm-test/node_modules/osrm
node-pre-gyp ERR! node -v v8.7.0
node-pre-gyp ERR! node-pre-gyp -v v0.6.38
node-pre-gyp ERR! not ok
403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/osrm/v5.13.0/Release/node-v57-linux-x64.tar.gz
~/cardash/osrm-test/node_modules/osrm/build ~/cardash/osrm-test/node_modules/osrm
CMake Error: The source directory "/home/damocles/cardash/osrm-test/node_modules/osrm" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
npm WARN osrm-test@1.0.0 No description
npm WARN osrm-test@1.0.0 No repository field.
npm WARN osrm-test@1.0.0 license should be a valid SPDX license expression

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! osrm@5.13.0 install: `node-pre-gyp install --fallback-to-build=false || ./scripts/node_install.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the osrm@5.13.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/damocles/.npm/_logs/2017-11-09T22_41_17_156Z-debug.log
```

It's not a huge problem as I can just use the C++ repo and communicate with it via the REST API, but there are probably many potential users who would stop at this point.

**Comment by danpat** [comment]
@dfellis No, we haven't published binaries for Node 8 yet - only Node 4 and 6.

We *just* added CI jobs for Node 8 support - the binary part will be published as part of the *next* OSRM release (5.14), but isn't part of 5.13.

**Comment by dfellis** [comment]
Ahh, gotcha. Well, probably better to use the REST API, anyways, so there's nothing preventing security updates from being applied on either side.
